index.html

```diff
@@ -37,6 +37,128 @@ <h1 class="name">
 			<div class="bottom_geometry"></div>
 		</section>
 
+		<section class="project_section">
```

Comment:
- [ ] The original design has 6 projects card. You have only 2 projects. Please add 4 projects as an original design.
- [ ] Project section has to same with the main section. To stick to the design please fix it. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20tests%2C%20margins)%20using%20the%20templates%20in%20Figma.)
![image](https://user-images.githubusercontent.com/38635158/148408072-e0f19ed4-f6b8-4aa8-982a-d55a95bc3943.png)

____

### ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/148407811-88c9bd60-092f-4e9f-8267-4fa03bae4836.png)